proposals/reduce_spatial.json

```diff
@@ -0,0 +1,79 @@
+{
+    "id": "reduce_spatial",
+    "summary": "Reduce horizontal spatial dimensions",
+    "description": "Applies a unary reducer to a data cube dimension by collapsing all the pixel values along the horizontal spatial dimensions (i.e. axes `x` and `y`) into an output value computed by the reducer. The horizontal spatial dimensions are dropped.\n\nThis process passes a list of values to the reducer. The list of values has an undefined order, therefore processes such as ``last()`` and ``first()`` that depend on the order of the values will lead to unpredictable results.\n\nIn contrast to this process, ``reduce_spatial_binary()`` passes two values, which may be better suited especially for UDFs in case the number of values gets too large to be processed at once. An aggregation over certain spatial areas can be computed with the process ``aggregate_spatial()``.",
```

Comment:
> Applies a unary reducer to a data cube dimension by

I think  "dimension" should not be there
(the process tackles two dimensions at same time)

Reply:
Yes, true. Removed.